src/hub/interfaces/IHoldings.sol

```diff
@@ -15,13 +15,23 @@ struct Holding {
     bool isLiability;
 }
 
+struct CreateHolding {
+    AccountId accountId;
+    uint8 kind;
+}
```

Comment:
Extracting the `kind` from the account means that now the `Accounting` stores a plain Account without kind information. That was a requirement if it had not changed

Reply:
The requirement is now that multiple `kind`s can write to the same `Account` in the `Accounting` contract, so the `kind` can't be embedded in the `accountId` anymore.

Reply:
That's weird. Is there any thread or issue with that requirement definition? The above would mean that asset & equity accounts can be mixed in the accounting.

Just to double check (from https://www.notion.so/Fix-Accounts-in-createHolding-1c02eac24e1780f0b283fa4f5bc6c7e2?pvs=4#1c02eac24e1780ef90e0eea46b34bde6):

> re-use an existing AccountType.Equity account for all assets

This means that several assets can have the same kind account